.github/actions/run-workspace-tests/action.yml

```diff
@@ -57,10 +57,11 @@ runs:
         cache-to: type=gha,mode=max
         build-args: BASE_IMAGE=${{ steps.ecr-login.outputs.registry }}/${{ inputs.ugbio_base_image }}
 
-    - name: Run tests in the container
+    - name: Run python tests (if exist) in the container
+      working-directory: src/${{ inputs.workspace-folder }}
       shell: bash
-      run: |
-        docker run --rm -v ${{ github.workspace }}/src:/workdir/src ${{ inputs.workspace-folder }} pytest /workdir/src/${{ inputs.workspace-folder }}
+      run: docker run --rm -v .:/workdir/src ${{ inputs.workspace-folder }} run_tests /workdir/src
```

Comment:
Why do you need the volume mapping?
It wouldn't work with:
docker run --rm ${{ inputs.workspace-folder }} run_tests

Reply:
can't work. see: https://github.com/Ultimagen/ugbio-utils/actions/runs/11040725852